features/margin.yml

```
@@ -0,0 +1,15 @@
+name: margin
+description: Margins define space surrounding a box. The `margin` CSS property provides a shorthand for `margin-top`, `margin-right`, `margin-bottom`, and `margin-left`.
+spec: https://drafts.csswg.org/css-box-4/
```

Ideally, there's a more specific anchor we can link to.

```suggestion
spec: https://drafts.csswg.org/css-box-4/
```

https://drafts.csswg.org/css-box-4/#margins